### ISSUE TITLE
task.gpx file invalid

### ISSUE THREAD

**Issue by Locke** [hotosm/osm-tasking-manager2#748]
The gpx file produced following the "Tip: Download the following .gpx file and load it in JOSM in order to see the current task boundary" hyperlink fails to validate:

```
$ wget http://tasks.hotosm.org/project/1096/task/3.gpx
$ xmllint --noout --schema http://www.topografix.com/GPX/1/1/gpx.xsd /tmp/3.gpx
/tmp/3.gpx:3: element gpx: Schemas validity error : Element '{http://www.topografix.com/GPX/1/1}gpx': The attribute 'version' is required but missing.
/tmp/3.gpx:3: element gpx: Schemas validity error : Element '{http://www.topografix.com/GPX/1/1}gpx': The attribute 'creator' is required but missing.
/tmp/3.gpx fails to validate
```

I also can't open these gpx files in josm anymore (opening fails with the message `org.xml.sax.SAXException: Parse error: invalid document structure for GPX document.`, this is how I discovered this issue). It worked some time ago, but I think josm may got a bit more restrictive in a recent version.

I think fixing this should be as easy as adding ` version="1.1" creator="HOT Tasking Manager"` to the root tag in /osmtm/templates/task.gpx.mako (adding this to the downloaded file works for xmllint and josm).

**Comment by pgiraud** [comment]
Thanks for the report.

**Comment by ethan-nelson** [comment]
Fixed via #764
